scripts/astyle.sh

```diff
@@ -27,20 +27,28 @@ fi
 
 set -e
 
-MY_ARTISTIC_STYLE_OPTIONS="\
---style=allman \
---preserve-date \
---indent-preprocessor \
---convert-tabs \
---indent=spaces=2 \
---indent-classes \
---indent-labels \
---indent-namespaces \
---indent-switches \
+MY_ARTISTIC_STYLE_OPTIONS=" \
```

Comment:
Should we merge this file already?
If you are afraid of merge/rebase problems, let's wait ;)

Reply:
no, you take all or nothing ;)